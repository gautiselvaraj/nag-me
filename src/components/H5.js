import React from 'react';
import styled from 'styled-components';

const Heading = styled.h5`
  font-size: 1rem;
  margin-top: 0.25rem;
  margin-bottom: 0.25rem;
`;

export default ({ children, ...otherProps }) => (
  <Heading {...otherProps}>{children}</Heading>
);
