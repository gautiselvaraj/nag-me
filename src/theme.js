export default {
  main:           '#214154',
  mainDarken:     '#083148',
  mainLighten:    '#215a7b',
  black:          '#111111',
  white:          '#f6f6f6',
  greyLightest:   '#efefef',
  greyLighter:    '#dddddd',
  greyLight:      '#a5a5a5',
  grey:           '#898989',
  greyDark:       '#777777',
  greyDarker:     '#595959',
  greyDarkest:    '#222',
}
