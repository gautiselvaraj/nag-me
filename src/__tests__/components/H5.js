import React from 'react';
import { mount } from 'enzyme';
import H5 from '../../components/H5';
import theme from '../../theme';
import 'jest-styled-components';

describe('<H5 />', () => {
  it('should match snapshot and have correct text & element', () => {
    const h5 = mount(<H5 theme={theme}>Heading 5</H5>);
    expect(h5).toMatchSnapshot();
    expect(h5.text()).toBe('Heading 5');
    expect(h5.find('h5').length).toBe(1);
  });
});
