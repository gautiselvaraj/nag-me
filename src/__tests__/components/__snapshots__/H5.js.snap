// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<H5 /> should match snapshot and have correct text & element 1`] = `
.c0 {
  font-size: 1rem;
  margin-top: 0.25rem;
  margin-bottom: 0.25rem;
}

<Component
  theme={
    Object {
      "black": "#111111",
      "grey": "#898989",
      "greyDark": "#777777",
      "greyDarker": "#595959",
      "greyDarkest": "#222",
      "greyLight": "#a5a5a5",
      "greyLighter": "#dddddd",
      "greyLightest": "#efefef",
      "main": "#214154",
      "mainDarken": "#08283B",
      "mainLighten": "#3B5B6E",
      "white": "#f6f6f6",
    }
  }
>
  <styled.h5
    theme={
      Object {
        "black": "#111111",
        "grey": "#898989",
        "greyDark": "#777777",
        "greyDarker": "#595959",
        "greyDarkest": "#222",
        "greyLight": "#a5a5a5",
        "greyLighter": "#dddddd",
        "greyLightest": "#efefef",
        "main": "#214154",
        "mainDarken": "#08283B",
        "mainLighten": "#3B5B6E",
        "white": "#f6f6f6",
      }
    }
  >
    <h5
      className="c0"
    >
      Heading 5
    </h5>
  </styled.h5>
</Component>
`;
